3-enrich/stream-enrich/kinesis/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/sinks/KinesisSink.scala

```diff
@@ -41,12 +41,7 @@ import scalatracker.Tracker
 
 /** KinesisSink companion object with factory method */
 object KinesisSink {
-  def createAndInitialize(
-    kinesisConfig: Kinesis,
-    bufferConfig: BufferConfig,
-    streamName: String,
-    tracker: Option[Tracker]
-  ): \/[String, KinesisSink] = for {
+  def validate(kinesisConfig: Kinesis, streamName: String): \/[String, Boolean] = for {
```

Comment:
For me it looks like `\/[String, Unit]` rather than `Boolean` - it'll never return `false`.